packages/rocketchat-lib/startup/defaultRoomTypes.js

```diff
@@ -64,8 +64,7 @@ RocketChat.roomTypes.add('d', 20, {
 	},
 
 	roomName(roomData) {
-		const room = ChatSubscription.findOne({ rid: roomData._id }, { fields: { name: 1 } });
-		return room && room.name;
+		return ChatSubscription.findOne({ rid: roomData._id }, { fields: { name: 1, fname: 1 } });
```

Comment:
`roomData` or the returned result of `ChatSubscription`? Is there a case where it might be undefined?